lib/compress.js

```diff
@@ -244,7 +244,11 @@ merge(Compressor.prototype, {
                 }
                 if (node instanceof AST_ForIn) {
                     if (node.init instanceof AST_SymbolRef) {
-                        node.init.definition().fixed = false;
+                        var d = node.init.definition();
+                        d.references.push(node.init);
+                        d.fixed = false;
```

Comment:
Just curious why the symbol ref logic here differs somewhat from:

https://github.com/alexlamsl/UglifyJS2/blob/274dd22340536c07ed354957ffa1b82a88fc9773/lib/compress.js#L195-L197